packages/melos/lib/src/commands/bootstrap.dart

```diff
@@ -54,34 +56,44 @@ mixin _BootstrapMixin on _CleanMixin {
                 devDependencies: bootstrapCommandConfig.devDependencies,
               );
             }).drain<void>();
+
+            logger
+              ..child(successLabel, prefix: '> ')
+              ..newLine();
```

Comment:
Probably enough to just write out the success after the full command is done, like it was before, or does it give anything to print success twice?

Reply:
I think there is a mix of terminology here, the command I'm thinking of as `bootstrap` and bootstrap command contains two steps, updating shared dependencies and updating the local linking, and right now it prints success after both steps (which might make sense, not sure)? Because if `skipLinking` is false it will also print success here:
https://github.com/invertase/melos/pull/684/files#diff-b5438310509b0fb6f04ef9776c2b9d3ec2a18fe0e0bc7165cdda954148ad9a99R76-R78

Reply:
Yes, but that success will point to the `Updating common dependencies` step and not the `Run pub get` step. And also Yes it makes sense to have two success log as they are separate steps.

### Logs with `--skip-linking` disabled
- Single success (Old)
```zsh
Running "dart pub get" in workspace packages...
  melos
  └> Updated environment
  melos_workspace
  └> Updated environment
  conventional_commit
  └> Updated environment
  ✓ melos_workspace
    └> .
  ✓ conventional_commit
    └> packages/conventional_commit
  ✓ melos
    └> packages/melos
  > SUCCESS
```
- Multiple success (New)
```zsh
Updating common dependencies in workspace packages...
  ✓ melos
    └> Updated environment
  ✓ conventional_commit
    └> Updated environment
  ✓ melos_workspace
    └> Updated environment
  > SUCCESS

Running "dart pub get" in workspace packages...
  ✓ melos_workspace
    └> .
  ✓ conventional_commit
    └> packages/conventional_commit
  ✓ melos
    └> packages/melos
  > SUCCESS
```

### Log with `--skip-linking` enabled
- Single success (Old)
```zsh
Updating common dependencies in workspace packages...
  melos
  └> Updated environment
  melos_workspace
  └> Updated environment
  conventional_commit
  └> Updated environment
  > SUCCESS
```

- Multiple success (New)
```zsh
Updating common dependencies in workspace packages...
  ✓ melos_workspace
    └> Updated environment
  ✓ conventional_commit
    └> Updated environment
  ✓ melos
    └> Updated environment
  > SUCCESS
```

If you can see, it won't make much difference in case `--skip-linking` enabled but the commands looks congested when `--skip-linking` is disabled with single success log.

Reply:
I agree it looks better :+1: